mysql.py

```diff
@@ -471,17 +481,44 @@ def fetch_mysql_process_states(conn):
 
 
 def fetch_mysql_variables(conn):
+    """ Fetch defined MySQL variables """
     global MYSQL_VARS
     result = mysql_query(conn, 'SHOW GLOBAL VARIABLES')
     variables = {}
     for row in result.fetchall():
         if row['Variable_name'] in MYSQL_VARS:
-                variables[row['Variable_name']] = row['Value']
+            variables[row['Variable_name']] = row['Value']
 
     return variables
 
 
+def is_ps_enabled(conn):
+    """ Check is performance_schema is enabled """
+    result = mysql_query(conn, 'SHOW GLOBAL VARIABLES LIKE "performance_schema"')
+    row = result.fetchone()
+    return bool(row['Value'] == 'ON')
+
+
+def fetch_connections_per_account(conn):
+    """ Fetch number of connections per account """
+    queries = {}
+    try:
+        result = mysql_query(conn, """
+		        SELECT user, current_connections
```

Comment:
this query returns one line per user, host couple and user can be null.
i think we should aggregate the results to have all the data :

```
SELECT user, sum(current_connections) as `current_connections`
FROM performance_schema.accounts
WHERE user is not null
GROUP BY user
```

Reply:
Indeed